examples/matroska_remux.rs

```diff
@@ -7,22 +7,23 @@ use matroska::{demuxer::MKV_DESC, muxer::MkvMuxer};
 use std::{fs::File, io::Cursor, sync::Arc};
 
 fn main() {
-    pretty_env_logger::init();
```

Comment:
Might be a good idea to spin out this change and make the remuxer take 2 arguments.